Index: media/libstagefright/ACodec.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- media/libstagefright/ACodec.cpp	(revision 4379388b9c6ef3304df7149aaa9663663ed66ca3)
+++ media/libstagefright/ACodec.cpp	(revision 72c738adba966934f697396b97f222bee84e4997)
@@ -1184,7 +1184,17 @@
     // 2. try to allocate two (2) additional buffers to reduce starvation from
     //    the consumer
     //    plus an extra buffer to account for incorrect minUndequeuedBufs
+#ifdef BOARD_CANT_REALLOCATE_OMX_BUFFERS
+    // Some devices don't like to set OMX_IndexParamPortDefinition at this
+    // point (even with an unmodified def), so skip it if possible.
+    // This check was present in KitKat.
+    if (def.nBufferCountActual < def.nBufferCountMin + *minUndequeuedBuffers) {
+#endif
+#ifdef STE_HARDWARE
+    for (OMX_U32 extraBuffers = 1; /* condition inside loop */; extraBuffers--) {
+#else
     for (OMX_U32 extraBuffers = 2 + 1; /* condition inside loop */; extraBuffers--) {
+#endif
         OMX_U32 newBufferCount =
             def.nBufferCountMin + *minUndequeuedBuffers + extraBuffers;
         def.nBufferCountActual = newBufferCount;
@@ -1203,6 +1213,9 @@
             return err;
         }
     }
+#ifdef BOARD_CANT_REALLOCATE_OMX_BUFFERS
+    }
+#endif
 
     err = native_window_set_buffer_count(
             mNativeWindow.get(), def.nBufferCountActual);
Index: media/libstagefright/Android.bp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- media/libstagefright/Android.bp	(revision 4379388b9c6ef3304df7149aaa9663663ed66ca3)
+++ media/libstagefright/Android.bp	(revision 72c738adba966934f697396b97f222bee84e4997)
@@ -202,6 +202,7 @@
         "-Wno-multichar",
         "-Wno-error=deprecated-declarations",
         "-Wall",
+        "-DBOARD_CANT_REALLOCATE_OMX_BUFFERS",
     ],
 
     version_script: "exports.lds",
